modules/navier_stokes/src/fvkernels/INSFVMomentumDiffusion.C

```diff
@@ -28,6 +30,31 @@ INSFVMomentumDiffusion::validParams()
 INSFVMomentumDiffusion::INSFVMomentumDiffusion(const InputParameters & params)
   : INSFVFluxKernel(params), _mu(getFunctor<ADReal>(NS::mu))
 {
+  if ((_var.faceInterpolationMethod() == Moose::FV::InterpMethod::SkewCorrectedAverage) &&
+      (_tid == 0))
+  {
+    auto & factory = _app.getFactory();
+
+    auto rm_params = factory.getValidParams("ElementSideNeighborLayers");
+
+    rm_params.set<std::string>("for_whom") = name();
+    rm_params.set<MooseMesh *>("mesh") = &const_cast<MooseMesh &>(_mesh);
+    rm_params.set<Moose::RelationshipManagerType>("rm_type") =
+        Moose::RelationshipManagerType::GEOMETRIC | Moose::RelationshipManagerType::ALGEBRAIC |
+        Moose::RelationshipManagerType::COUPLING;
+    FVKernel::setRMParams(
+        _pars, rm_params, std::max((unsigned short)(3), _pars.get<unsigned short>("ghost_layers")));
```

Comment:
can we have a default instead?
If I m not doing skew correction (which vtb likely wont do, we have pretty meshes), I d like to stick with 2

Reply:
This only triggers if there is skew correction. If you do not have skew correction, then you will stay with 2